shared/domains/chelonia/chelonia.js

```diff
@@ -91,7 +208,8 @@ export default (sbp('sbp/selectors/register', {
         handleEventError: null, // (e: Error, message: GIMessage) => {}
         syncContractError: null, // (e: Error, contractID: string) => {}
         pubsubError: null // (e:Error, socket: Socket)
-      }
+      },
+      transientSecretKeys: {}
```

Comment:
Should this be added here to the `config` where it is more accessible to the outside code? Doesn't seem like a "config" thing.

Could be added directly to `this` instead?

Reply:
We decided on this after a call to be able to use `sbp('chelonia/configure', { transientSecretKeys })`, in an effort to mostly deprecate `withEnv` (or reduce its use at least).

It *could* be added to `this`, but it needs creating a new selector that can act in a similar way to `config`.